### PR TITLE
fix GCP OEM functionality

### PR DESCRIPTION
**Pull request by jepio**
# fix GCP OEM functionality

Since we dropped python2 from our tree, the GCP OEM container image has been broken. This is because we explicitly removed/masked all python3 packages in the oem-aci profile. After restoring python in the container image, it turns out more dependencies were missing and we need to update the compute-engine-packages to work with python 3.9.

Requires: https://github.com/flatcar-linux/portage-stable/pull/321
Related: https://github.com/flatcar-linux/Flatcar/issues/714

## How to use

```
./build_packages
./build_image
./image_to_vm.sh --format=gce
```

## Testing done

Testing this together with some semi-related commits here: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5373/cldsv/.

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

**Comment by pothos** [comment]
Thanks, looks good. I really think this is the kind of functionality that should be tested by kola because it's something not developed by an upstream project but custom to Flatcar.

**Comment by jepio** [comment]
Agree - looking into adding a test for this.

**Comment by jepio** [comment]
This is now covered by tests, could you have a second look @pothos ?

File: changelog/bugfixes/2022-04-12-fix-gcp-oem-service.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- Restored oem-gce.service functionality on GCP (TODO)
```
Comment (pothos) [inline changelog/bugfixes/2022-04-12-fix-gcp-oem-service.md:1]:
```suggestion
- Restored oem-gce.service functionality on GCP ([PR#1813](https://github.com/flatcar-linux/coreos-overlay/pull/1813))
```

File: changelog/updates/2022-04-12-google-compute-engine.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- google compute-image-packages ([20190124](https://github.com/GoogleCloudPlatform/compute-image-packages/releases/tag/20190124))
```
Comment (pothos) [inline changelog/updates/2022-04-12-google-compute-engine.md:1]:
```suggestion
- GCE: google compute-image-packages ([20190124](https://github.com/GoogleCloudPlatform/compute-image-packages/releases/tag/20190124))
```

**Review by pothos — APPROVED** [review]
Thanks, looks good, just some suggestions above

**Comment by jepio** [comment]
Fixed up the changelog, now going in for the cherry-pick.

**Pull request merged**